frontend/src/services/connectionTypesService.ts

```diff
@@ -0,0 +1,78 @@
+import axios from '~/utilities/axios';
+import { ResponseStatus } from '~/types';
+import {
+  ConnectionTypeConfigMap,
+  ConnectionTypeConfigMapObj,
+} from '~/concepts/connectionTypes/types';
+import {
+  toConnectionTypeConfigMap,
+  toConnectionTypeConfigMapObj,
+} from '~/concepts/connectionTypes/utils';
+
+export const fetchConnectionTypes = (): Promise<ConnectionTypeConfigMapObj[]> => {
+  const url = `/api/connection-types`;
+  return axios
+    .get(url)
+    .then((response) =>
+      response.data.map((cm: ConnectionTypeConfigMap) => toConnectionTypeConfigMapObj(cm)),
```

Comment:
I believe this will be inside `data.items`.
```suggestion
      response.data.items.map((cm: ConnectionTypeConfigMap) => toConnectionTypeConfigMapObj(cm)),
```

Reply:
nm! the backend point returns a list